math/calculator.cpp

```diff
@@ -34,6 +34,12 @@ int main() {
     Node* new_node = new Node();
     requestData(new_node);
 
+    /**
+     * Release allocated memory used in `new_node`.
+     */
+
+    delete[] new_node;
```

Comment:
`delete[]` operator is for arrays
`delete` operator is for single arrays
[C++ 101](https://docs.microsoft.com/en-us/cpp/cpp/delete-operator-cpp?view=vs-2019)

Reply:
Fixed.